core/src/formatters/nova.ts

```diff
@@ -56,17 +51,26 @@ export async function formatNovaPrompt(segments: PromptSegment[], schema?: JSONS
     for (const segment of segments) {
 
         const parts: NovaMessagePart[] = [];
-        /*
         if (segment.files) for (const f of segment.files) {
             //TODO type: 'image' -> detect from f.mime_type
```

Comment:
done?

Reply:
Thanks for noticing, comments cleaned up.